Jenkinsfile-Release

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc. All rights reserved.
+ * Includes the third-party code listed at http://links.sonatype.com/products/nexus/attributions.
+ * "Sonatype" is a trademark of Sonatype, Inc.
+ */
+@Library(['private-pipeline-library', 'jenkins-shared', 'int-jenkins-shared']) _
+
+dockerizedBuildPipeline(
+  prepare: {
+    githubStatusUpdate('pending')
+  },
+  buildAndTest: {
+    sh './build.sh'
+  },
+  skipVulnerabilityScan: true,
+  archiveArtifacts: 'docs/*',
+  testResults: [],
+  deployCondition: { true },
+  deploy: {
+    final branch = gitBranch(env)
+
+    sh """
+      git config --global user.email 'sonatype-ci@sonatype.com'
+      git config --global user.name 'Sonatype CI'
+
+      git checkout ${branch}
+      git commit -a -m 'Release Update'
```

Comment:
no push of this commit?

Reply:
oops, this was still some observable debugging. "push" coming right up.